### PR TITLE
fix(SF.Issue.972): Programs and Features icon in Windows

### PR DESCRIPTION
**Pull request by stef145g**
Ref: https://sourceforge.net/p/moneymanagerex/bugs/972/

---
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/1011)

### CODE REVIEW

File: setup/win/mmex_x64_setup.iss [hunk 1]
```diff
@@ -77,29 +80,34 @@ ArchitecturesAllowed=x64
...
-Name: norwegian; MessagesFile: compiler:Languages\Norwegian.isl; 
-Name: polish; MessagesFile: compiler:Languages\Polish.isl; 
-Name: portuguese; MessagesFile: compiler:Languages\Portuguese.isl; 
-Name: serbiancyrillic; MessagesFile: compiler:Languages\SerbianCyrillic.isl; 
-Name: serbianlatin; MessagesFile: compiler:Languages\SerbianLatin.isl; 
-Name: slovenian; MessagesFile: compiler:Languages\Slovenian.isl; 
-Name: spanish; MessagesFile: compiler:Languages\Spanish.isl; 
-Name: ukrainian; MessagesFile: compiler:Languages\Ukrainian.isl; 
+Name: "english"; MessagesFile: "compiler:Default.isl"
+Name: "basque"; MessagesFile: "compiler:Languages\Basque.isl"
```
Comment (vomikan) [inline setup/win/mmex_x64_setup.iss]: Seems new languages missing 
https://github.com/moneymanagerex/moneymanagerex/commit/9018698033e91333deee6a3c80e346ed1d6f5a74

Comment (vomikan) [inline setup/win/mmex_x64_setup.iss]: Abkhazian Albanian Burmese Catalan Finnish Korean Malayalam Persian Thai

Comment (vomikan) [inline setup/win/mmex_x64_setup.iss]: Catalan.isl is present in Inno Setup +Finnish

Comment (stef145g) [inline setup/win/mmex_x64_setup.iss]: The languages here are the ones that InnoSetup provides at startup. Any missing ones have not been provided.
5 extra languages have been provided.

**Pull request merged**